views/js/qtiCreator/editor/widgetToolbar.js

```diff
@@ -387,7 +387,7 @@ define([
                 toolbar = buildToolbar(buttonConfig, target, tlbWrapper);
 
                 if(barConfig.title){
-                    title = $('<span>', {text : barConfig.title, 'class' : 'tlb-title', title : barConfig.title});
+                    title = $('<span>', { text : __(barConfig.title), 'class' : 'tlb-title', title : __(barConfig.title) });
```

Comment:
Translations must be done from a static text. The use of variables prevents the i18n scanner to work properly, as it won't be able to list the input.

Since the value comes from the config, please apply the change to the consumer instead.
Moreover, since the text to translate is not presented in the code, but will probably bee added manually in the PO file, the next round of i18n scan will remove it.